packages/syft/src/syft/core/node/common/client.py

```diff
@@ -510,7 +510,9 @@ def get(self, key: Union[str, int, UID]) -> Pointer:
                 address=self.client.address, reply_to=self.client.address, obj_id=key
             )
             results = getattr(
-                self.client.send_immediate_msg_with_reply(msg=msg), "results", None
+                self.client.send_immediate_msg_with_reply(msg=msg, timeout=600),
```

Comment:
@iamtrask, it's 30 seconds by default. But it may be not be ideal for get or upload data (since size of data can be large and could take time to download). I move these timeouts to constant variables to its easy to read and manipulate those values.